lib/passport-wsfed-saml2/saml.js

```diff
@@ -97,6 +93,11 @@ SAML.prototype.validateSignature = function (xml, options, callback) {
     return callback(e);
   }
 
+  if (self.options.checkCertExpiration && !self.validateCertExpiration(xml)) {
+    valid = false;
```

Comment:
Does this serve any purpose if you return on the next line?

Reply:
No, that's vestigial, thanks. removed.